_posts/2018-01-17-what-is-on-my-mac-2018.md

```diff
@@ -0,0 +1,79 @@
+
+- [**Fantastical**][5830-0012]:
+The Calendar app on the Mac is a mess (and on iOS, but that's a separate article). I used BusyCal for a while, and it was ok. In 2012, I didn't feel their app was *that* much better. When Fantastical came about, I fell in love. At the time, just a simple menu bar app that listed the day's events. Fast forward almost six years and the Flexibits team continue to make Fantastical even better.
+
+- [**Gifox**](https://gifox.io/):
+Gifox is a handy menu bar app that records your screen and creates a GIF of that recording. This app has been insanely useful in GitHub pull requests. I'll often record an interaction for the person reviewing the PR. This has proved to be helpful to ensure things are working as they're supposed to.
+
+- [**Moom**](https://manytricks.com/moom/):
+Window management on the Mac is terrible. I never understood what the little green button did, then they changed it to the fullscreen button. As someone who works with multiple displays, having my windows organized in the right way is important. Moom even lets me send windows to a different display with a quick shortcut.
+
+- [**Bartender**](https://www.macbartender.com/):
+If you're like me, you have many menu bar apps. That also means that at some point you run out of space. Even if I wasn't running out of space though, I like my menu bar to only have the critical apps or functions I need. Bartender has helped me keep things tidy.
+
+- [**Frank DeLoupe**](http://jumpzero.com/frank/):
+When I'm writing the codes, I find myself needing to sample colors. This nifty menu bar app (see a pattern here?) allows you to summon it with a keyboard shortcut and then puts the color on your clipboard. Frank gives you sixteen different options for what syntax you want that color in. Crazy awesome.
+
+- [**Tweetbot**](https://tapbots.com/tweetbot/mac/):
+Tweetbot is easily the biggest reason I still use Twitter. I don't see any of the weird behavior present in the native Twitter apps. The only thing I miss because I use Tweetbot are polls. Oh well.
+
+- [**Kuvva**](https://www.kuvva.com/):
+If you enjoy wallpapers, Kuvva is a perfect menubar app that changes your wallpaper periodically. You can set it to change your wallpaper daily or even every thirty minutes. [Rachel](https://twitter.com/missrachilli) was the one to tell me about this one.
+
+- [**EncryptMe**](https://encrypt.me/):
+I think it's a good idea to use a VPN if you're not on a secured network. Hell, it might even be a good idea to use a VPN even if you are. My tinfoil hat aside, EncryptMe (formerly Cloak) makes it incredibly easy to use a VPN and to give them money with the variety of pricing options. One of the many is bound to suit your needs.
```

Comment:
"Hell, it might even be a good idea to use a VPN even if you are."

Use of the word even twice.